src/main/java/com/fvv/bookstore/view/BookViewImpl.java

```diff
@@ -137,9 +129,13 @@ public void updateBook() {
 			e.printStackTrace();
```

Comment:
you can group this exception with the ControllerException since the implementation is the same. Ex catch (BookNotFoundException  | BookValidationException | ControllerException  e).....

Reply:
ok